pyproject.toml

```diff
@@ -79,6 +79,7 @@ ray = { version = "==2.10.0", optional = true, python = ">=3.8,<3.12" }
 requests = { version = "^2.31.0", optional = true }
 starlette = { version = "^0.31.0", optional = true }
 uvicorn = { version = "^0.23.0", extras = ["standard"], optional = true }
+sphinx-substitution-extensions = "2022.02.16"
```

Comment:
This should be a dev dependency.

Reply:
My bad! I have moved it to the dev dependencies.